### PR TITLE
Update maintainers and authors

### PR DESCRIPTION
**Pull request by RichardScothern**
All hail the new regime.

Signed-off-by: Richard Scothern <richard.scothern@gmail.com>

### CODE REVIEW

**Comment by BrianBland** [comment]
:+1:

**Comment by dmp42** [comment]
LGTM

**Comment by RichardScothern** [comment]
LGTM

**Comment by aaronlehmann** [comment]
LGTM

File: MAINTAINERS [hunk 1]
```diff
@@ -32,6 +32,11 @@
 	Email = "aaron.lehmann@docker.com"
 	GitHub = "aaronlehmann"
 
+	[people.brianbland]
```
Comment (stevvooe) [inline MAINTAINERS]: Grats! @BrianBland

**Pull request merged**